lib/src/test/java/vmath/Mat4Test.java

```diff
@@ -195,6 +195,24 @@ void givenInvalidIndices_whenGetIsCalled_thenThrowsException() {
         assertThrows(IndexOutOfBoundsException.class, () -> m.get(0, 4));
     }
 
+    @Test
+    void givenVec4_whenMultiplied_thenResultIsCorrect() {
+        Mat4 m = new Mat4(
+            1.0f, 2.0f, 3.0f, 4.0f,
+            5.0f, 6.0f, 7.0f, 8.0f,
+            9.0f, 10.0f, 11.0f, 12.0f,
+	    13.0f, 14.0f, 15.0f, 16.0f
```

Comment:
Please, fix the indentation

Reply:
> Please, fix the indentation

Could you please clarify what the problem with the indentation is? I checked and rechecked, and it seems consistent with the formatting of the rest of the code.

Reply:
<img width="610" alt="Screenshot 2024-01-06 at 11 29 35 AM" src="https://github.com/toksaitov/vmath/assets/97272/8fc73754-38ad-4a18-85b6-f6d0705446cd">

Take a look at lines 204 and 213.